### DISCUSSION TITLE
Bryan Suryajaya | brya

### DISCUSSION THREAD

**Discussion by milky-chance** [onlyphantom/llm-python#28] [Practicum: Generative AI]
Thanks for running the workshop Samuel! The exposure given to the technology is simply amazing. I gained a lot of valuable experience from this.
Hopefully you can help me improve upon this submission, as I believe that there are a lot of room for improvements.

https://colab.research.google.com/drive/1bh2J-Ft97f55YwtI70ppd1EFUC00BAlR?usp=sharing

**Comment by onlyphantom** [comment]
Hey! Well done! I like that you've attempted both Challenge 1 and 2 on one go too. Marking as Passed and will issue you a certificate soon! 

I'm continually updating the materials on [the Generative AI series at Sectors Docs](https://docs.sectors.app/recipes/generative-ai-python/02-tool-use) so I'd also recommend giving that a thorough read!


Sam
